trunk/SUAVE/Components/Energy/Converters/Rotor.py

```diff
@@ -664,36 +839,110 @@ def prop_vel_to_body(self):
         Properties Used:
         None
         """
-        
+
         body2propvel = self.body_to_prop_vel()
-        
+
         r = sp.spatial.transform.Rotation.from_matrix(body2propvel)
         r = r.inv()
         rot_mat = r.as_matrix()
 
         return rot_mat
 
+def compute_HFW_blade_velocities( prop ):
```

Comment:
This should be pulled out of this file and placed in Methods

Reply:
relocated to methods